schedule/yam/agama_profile.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: agama
+description: >
+  Perform interactive installation with agama.
+schedule:
+  - yam/agama/boot_agama
+  - yam/agama/import_agama_profile
+  - yam/agama/patch_agama_tests
+  - yam/agama/agama_bundle
+  - installation/grub_test
+  - installation/first_boot
+  - yam/validate/validate_product
+  - yam/validate/validate_user
```

Comment:
this is not needed, we need validation for encryption.

Reply:
Removed and added related modules